source/addonHandler/__init__.py

```diff
@@ -507,7 +510,7 @@ def loadModule(self, name: str) -> ModuleType:
 			sys.modules[parentModule.__name__] = parentModule
 		spec = importlib.machinery.PathFinder.find_spec(fullNameTop, [self.path])
 		if not spec:
-			return None
+			raise ModuleNotFoundError(importlib._bootstrap._ERR_MSG.format(name), name=name)

```

Comment:
considering this relies on an underscored part of python, can we expect stability here when updating python? It might be safer to use a custom message here.

Reply:
Good point. I changed this into an f-string.